boot-qemu.sh

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env bash
+
+# Root of the repo
+BASE=$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)
+
+
+# Prints an error message in bold red then exits
+function die() {
+    printf "\n\033[01;31m%s\033[0m\n" "${1}"
+    exit 1
+}
+
+
+# Check that a binary is found
+function checkbin() {
+    command -v "${1}" &>/dev/null || die "${1} could not be found, please install it!"
+}
+
+
+# Parse inputs to the script
+function parse_parameters() {
+    while (( ${#} )); do
+        case ${1} in
+            -a|--arch|--architecture)
+                shift
+                case ${1} in
+                    arm32_v5|arm32_v6|arm32_v7|arm64|mips|mipsel|ppc32|ppc64|ppc64le|x86_64) ARCH=${1} ;;
+                    *) die "Invalid --arch value '${1}'" ;;
```

Comment:
I'm not sure if ARCH list should be hardcoded like this, but seems okay for now.

Reply:
I am not really sure of a better way to do it other than this? This matches how it is handled in driver.sh (just error when ARCH is undefined).

Reply:
Was just thinking of a config-file based configuration. If config with such name exists, then it's considered a valid value. Anyway current approach should be fine for now, just my thoughts.